package.json

```diff
@@ -2,12 +2,12 @@
   "name": "@scratchaddons/contributors",
   "version": "1.0.0",
   "description": "",
-  "main": "src/index.js",
+  "main": "src/index.mjs",
   "type": "module",
   "scripts": {
-    "test": "node src/index.js",
-    "run": "node src/index.js",
-    "debug": "node src/index.js"
+    "test": "node src/index.mjs",
+    "run": "node src/index.mjs",
+    "debug": "node src/index.mjs"
   },
```

Comment:
the `type: module` makes the `.mjs` extension unnecessary

Reply:
But `"type": "module"` makes it a module, and so does `type="module"` on `<script>` tags in the browser

Reply:
> Except you need to use modules (thus with the .mjs extension) in the browser (this is a browser requirement) to use some features like top level await, which is quite convenient and that I chose to use.

Fact is, this script is ran on Node.js, so `.mjs` is unnecessary.

Reply:
> > Except you need to use modules (thus with the .mjs extension) in the browser (this is a browser requirement) to use some features like top level await, which is quite convenient and that I chose to use.
> 
> Fact is, this script runs on Node.js only, so `.mjs` is unnecessary.

Well, I don't know what I had i mind when writing my previous answer but yeah, this code is meant to run on node.

Anyway, to use top level await in node (14+), you still need to use ES modules. This can be achieved, as mentionned, in multiple ways:
* either by using the `.mjd` extension (to tell the module files appart from the regular JS),
* or by setting the `type` property to `module` in the `package.json`.

I chose to do both (which, by the way, doesn't hurt), to make it clear that we are actually dealing with modules. This way, you don't have to look in the `package.json` file for a detail you usually wouldn't pay attention to and that you probably would have missed. The `.mjs` extension makes it more noticeable. Also, since node and other services check the `package.json` file for several reasons, we might as well make it clear there too that we are working with ES modules.

Reply:
> either by using the `.mjd` extension (to tell the module files appart from the regular JS),

Isn't it `.mjs`?